.github/workflows/actions.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ['2.5', '2.6', '2.7', '3.0']
+        activerecord: ['4.2', '5.0', '5.1', '5.2']
+        task: ['bundle exec rake spec']
```

Comment:
```suggestion
```

Reply:
remove all mentions of task and hardcode spec
atm it runs `bundle exec rake bundle exec rake spec` ... idk how that ever worked :D